ACD-502 (#26)
Actindo-AG/Documentation: DataHub/Operation/04_ManageDashboards.md

  The *Edit tab* view is displayed.

  ![Tab](../../Assets/Screenshots/DataHub/Overview/Tab.png "[Tab]")

3. Click the [ADD ELEMENT] button in the upper right corner.    
  The *Dashlet picker* window is displayed.

  ![Dashlet picker](../../Assets/Screenshots/DataHub/Overview/DashletPicker.png "[Dashlet picker]")

4. Select the checkboxes of the dashlets you want to add to your dashboard. If you click the checkbox in the header, all dashlets in the list are selected.

  > [Info] For detailed information about the available dashlets, see [Dashlet picker](../UserInterface/01a_Dashboards.md#dashlet-picker).

5. Click the [ACCEPT] button in the bottom right corner of the window.   
  The selected dashlets have been added to the dashboard. The *Dashlet picker* window is closed.

6. Click the [SAVE] button.   
  The dashboard has been saved. The *Edit tab* view is closed.

  > [Info] If you want to change the dashlet positioning, see [Change the dashlets arrangement](#change-the-dashlets-arrangement).

### Change the viewport

Change the viewport to adjust the view of the dashboard to the currently used device.
The viewport is saved for a single dashboard.

> So you can create dashboards with different viewports.

Reviewer: Ich würde beide Sätze verbinden: The viewport is saved for a single dashboard, so you can create dashboards with different viewports.

- At least one dashlet has been added to the dashboard, see [Add a dashlet](#add-a-dashlet).

#### Procedure

*DataHub > Overview > Tab DASHBOARDS*

![Tab with dashlets](../../Assets/Screenshots/DataHub/Overview/TabDashlets.png "[[Tab with dashlets]")

1. Select the tab of the dashboard to which you want to change the positioning of dashlets.   
  The selected dashboard tab is displayed.

2. Click the ![Settings](../../Assets/Icons/Settings01.png "[Tool]") (Settings) button in the upper right corner.   
  The *Edit tab* view is displayed.

  ![Edit tab](../../Assets/Screenshots/DataHub/Overview/EditTab.png "[Edit tab]")

3. Hover over a dashlet until the cursor icon changes to the cross arrows cursor.

4. Click the dashlet and move it on the desired position on the dashboard using drag and drop. When moving the dashlet across the dashboard, the possible positions on the predefined grid are highlighted.

  ![Move dashlet](../../Assets/Screenshots/DataHub/Overview/MoveDashlet.png "[Move dashlet]")

5. Click the [SAVE] button.   
  Any changes to the dashboard have been saved. The *Edit tab* view is closed.

### Remove a dashlet

> Remove a dashlet from a dashboard if it is not needed for the purposes or was added accidentally to the dashboard.

Reviewer: ```suggestion
Remove a dashlet from a dashboard if it is not needed for the purposes or has been added accidentally to the dashboard.
```

#### Prerequisites

- A dashboard tab has been created, see [Create a dashboard](#create-a-dashboard).
- At least one dashlet has been added to the dashboard, see [Add a dashlet](#add-a-dashlet).

#### Procedure

*DataHub > Overview > Tab DASHBOARDS*

![Tab with dashlets](../../Assets/Screenshots/DataHub/Overview/TabDashlets.png "[[Tab with dashlets]")

> 1. Select the tab of the dashboard to which you want to remove a dashlet.   

Reviewer: ```suggestion
1. Select the tab of the dashboard from which you want to remove a dashlet.   
```

  The selected dashboard tab is displayed.

2. Click the ![Settings](../../Assets/Icons/Settings01.png "[Settings]") (Settings) button in the upper right corner.    
  The *Edit tab* view is displayed.

  ![Edit tab](../../Assets/Screenshots/DataHub/Overview/EditTab.png "[EditTab]")

> 3. Click the ![Remove](../../Assets/Icons/Cross03.png "[Remove]") (Remove) button in the upper right corner of the dashlet you want to delete.    

Reviewer: Haben wir dieses Button (cross / x) waonders nicht Delete genannt?

> 3. Click the ![Remove](../../Assets/Icons/Cross03.png "[Remove]") (Remove) button in the upper right corner of the dashlet you want to delete.    

Reviewer: Remove vs. Delete (cross / X Zeichen) -> Ich habe kein Problem mit remove und delete, aber wollen wir sie vielleicht vereinheitlichen? Vgl. unten Delete a dashboard.

  The dashlet has been removed from the dashboard.

  > [Info] The removal cannot be undone. To recover a dashlet, you have to add it again, see [Add a dashlet](#add-a-dashlet).

4. Click the [SAVE] button.   
  Any changes to the dashboard have been saved. The *Edit tab* view is closed.

## Delete a dashboard

Delete a dashboard that is no longer needed to keep track of the other dashboards.

#### Prerequisites

A dashboard tab has been created, see [Create a dashboard](#create-a-dashboard).

#### Procedure

*DataHub > Overview > Tab DASHBOARDS*

![Multiple dashboards](../../Assets/Screenshots/DataHub/Overview/MultipleDashboards.png "[Multiple dashboards]")

1. Click the ![Delete](../../Assets/Icons/Cross02.png "[Delete]") (Delete) button on the dashboard tab you want to delete.   
  The *Confirm tab deletion* window is displayed.

  ![Confirm tab deletion](../../Assets/Screenshots/DataHub/Overview/ConfirmTabDeletion.png "[Confirm tab deletion]")

2. Click the [ACCEPT] button in the bottom right corner.   

>   The *Confirm tab deletion* window is closed. The dashboard has been deleted.

Reviewer: ```suggestion
  The *Confirm tab deletion* window is closed. The dashboard tab has been deleted.
```

>   > [Info] Be aware that the dashboard cannot be recovered as the deletion process cannot be undone.

Reviewer: ```suggestion
  > [Info] Be aware that the dashboard tab cannot be recovered as the deletion process cannot be undone.
```
Dashboard oder dashboard tab???

Author: 1 tab = 1 dashboard
